### PR TITLE
variables changed of WPScan to lower case wpscan

### PR DESCRIPTION
**Pull request by shreya10-chavan**

### CODE REVIEW

File: src-tauri/tauri.conf.json [hunk 1]
```diff
@@ -293,8 +293,8 @@
                         "args": true
                     },
                     {
-                        "name": "WPScan",
-                        "cmd": "WPScan",
+                        "name": "wpscancan",
```
Comment (basoreeper3007) [inline src-tauri/tauri.conf.json:296]:
```suggestion
"name": "wpscan",

**Review by basoreeper3007 — CHANGES_REQUESTED** [review]

**Comment by michaeljpigott** [comment]
@shreya10-chavan can you please confirm that I can close this pull request because it was replaced by a subsequent one #860?

**Comment by shreya10-chavan** [comment]
Yes please. You can close it

**Comment by michaeljpigott** [comment]
Closing pull request because it was superseded by another one.

**Pull request closed without merging**